server/models/Profile.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+//add comment to repush on correct branch
+const profileSchema = new mongoose.Schema({
+  firstName: { type: String },
+  lastName: { type: String },
+  gender: { type: String },
+  birthdate: { type: Date },
+  email: { type: String },
+  phoneNumber: { type: String },
+  address: { type: String },
+  description: { type: String },
+  availability: {
```

Comment:
This design for availability may not fit our project use case but let's look at it on a later date 😄